httpclient5/src/main/java/org/apache/hc/client5/http/async/methods/IgnoreCompleteExceptonFutureCallback.java

```diff
@@ -30,6 +30,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
```

Comment:
@arturobernalg Please make this class extend `IgnoreCompleteExceptionFutureCallback`.

Reply:
> @arturobernalg Please make this class extend `IgnoreCompleteExceptionFutureCallback`.

Fixed al the remarks.
TY